site/content/building-apps-for-livechat/extending-livechat-ui.md

```diff
@@ -64,10 +68,59 @@ Fullscreen apps help to reduce _context switching_. Users no longer need to swit
 Some applications require settings configuration, especially when they use external services. After installing the app, users need to do the setup in order for the application to work.
 
 We provide a special place called Settings page, where you can embed a page with configuration. It's recommended to use it together with [Sign in with LiveChat](/docs/sign-in-with-livechat/).
 
-See how to setup a Settings page for your app in the [documentation](https://developers.livechatinc.com/docs/agent-app-widgets/).
\ No newline at end of file
+See how to setup a Settings page for your app in the [documentation](https://developers.livechatinc.com/docs/agent-app-widgets/).
+
+## Extending the Chat Widget
+
+### Mobile Chat Widgets
+Have you got a mobile app? You can still make use of LiveChat and improve your *mobile* user experience! All you need to do is install the mobile **Chat Widget** compatible with your app.
+
+- Mobile [Chat Widget for iOS](https://developers.livechatinc.com/docs/ios-widget/) apps
+- Mobile [Chat Widget for Android](https://developers.livechatinc.com/docs/android-widget/) apps
+- Mobile [Chat Widget for React Native](https://developers.livechatinc.com/docs/react-native-livechat/) apps 
+ 
+
+### Rich messages
+Sending a [Rich message](https://www.livechatinc.com/kb/rich-messages/) is the way to go when you want your reply to be more personal and visually attractive. Rich messages are more than just plain text. You can make use of images, buttons, quick replies, and cards as well. 
+
+Rich messages are the core feature of our **ChatBot** integration. If you're not familiar with ChatBot, [this article](https://www.livechatinc.com/kb/chatbots-explained/) will come in handy. Currently, they're available either with our ChatBot integration or [Beta API 3.0](https://developers.livechatinc.com/beta-docs/).
+
+
+### Moments
+[Chat Widget Moments](https://developers.livechatinc.com/docs/chat-widget-moments/) are webpages embedded in an iframe, displayed during a chat. 
+![Moment](moments.png)
+
+Thanks to **Moments**, customers can perform specific actions. For example, you could pick a date from a popping up calendar instead of typing it manually. Moments were created to solve complex problems easily and securely. You can think of them as an accessible API to integrate the **Chat Widget** with some other software. 
+
+
+### JavaScript API
+[**Chat Widget JavaScript API**](https://developers.livechatinc.com/docs/js-api/) is an easy yet powerful way to integrate your website or application with the **LiveChat** platform. 
+This is what e-commerce owners often use to pass visitor data or to track sales made via chat (read about [Sales tracker](https://www.livechatinc.com/kb/sales-tracker/) to learn more).
+
+The integration of your app with the **Chat Widget** can be multiform and contextual. To improve the customer support experience, you could add a _chat-with-customer-support_ button in those app locations, which trouble users the most. 
```

Comment:
Oprócz "locations which trouble users the most" dodałbym jeszcze przypadek dodania tego przycisku na stronie "contact", jako coś co powinno być standardem wg https://www.nngroup.com/articles/chat-ux/ : 
> 1.Place a Link to Chat on the Contact Us Page and Label it Chat

Reply:
Will do